src/ExponentialUtilities.jl

```diff
@@ -34,6 +35,8 @@ include("krylov_phiv_adaptive.jl")
 include("kiops.jl")
 include("StegrWork.jl")
 include("krylov_phiv_error_estimate.jl")
+# precompile script
+include("ExpUtil_precomp.jl")
```

Comment:
just name the file precompile.jl. No need to copy the package name in the package.